app/src/main/java/com/jienihou/outfitxchange/HomeScreen.kt

```diff
@@ -15,11 +23,52 @@ import androidx.fragment.app.Fragment
  * create an instance of this fragment.
  */
 class HomeScreen : Fragment() {
+    private lateinit var bottomNavigationView: BottomNavigationViewEx
+    private lateinit var categoryAdapter: CategoryRVAdapter
```

Comment:
this probably doesn't need to be late-init, you can probably initialize here:

`val categoryAdapter = CategoryAdapter()`